notebooks/supernova_model_sims.ipynb

```diff
@@ -2,79 +2,380 @@
  "cells": [
```

Comment:
<p>can you plot the light curve with earlier times (say phase -50 days) to see the light curves raising?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>